example/rcnn/demo.py

```diff
@@ -57,7 +57,7 @@ def demo_net(detector, image_name):
         cls_scores = cls_scores[keep]
         dets = np.hstack((cls_boxes, cls_scores[:, np.newaxis])).astype(np.float32)
         keep = nms(dets.astype(np.float32), NMS_THRESH)
-        all_boxes[cls_ind] = dets[keep, :]
+        all_boxes[cls_ind] = dets[keep, :] if dets.shape[0] != 0 else dets
```

Comment:
I also thought before, and there should be a zero vector in some rows when some instances are below thresh .But what about this kind of situation that `keep` has no instance? I just picked a picture then i encounter this error, a simple judgement modification helped me solve it.